developer-docs-site/docs/tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -719,6 +719,54 @@ The `peer_config.yaml` file will be created in your current working directory, w
 
 Move examples can be found in the [Move section](../../move/move-on-aptos/cli).
 
+You can also pass multiply-nested vector arguments, like in the `cli_args` example from [move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples):
```

Comment:
`multi-nested` vs `multiply-nested` sounds like you're multiplying :)

Reply:
Resolved in 329d1e5